pkg/memory-monitor/README.md

```diff
@@ -354,3 +354,44 @@ It will allocate 100Mb of memory and release it after user presses Enter.
 It's worth mentioning that if you run the tool from an ssh session, it will
 allocate memory in the ssh session cgroup, that is exactly a part of the eve
 cgroup. So, it will trigger the eve cgroup memory event.
+
+## PSI statistics collector
+
+There is a tool that can be used to collect the PSI statistics from the EVE.
+The PSI is a kernel feature that provides information about the pressure on the
+memory, CPU, and IO subsystems. In our case, we are interested in the memory
+pressure.
+
+The tool is built using the Makefile:
+
+```shell
+$ make statistics-collector
+```
+
+It will create a `statistics-collector` binary in the `bin` directory. The
+binary can be copied to the EVE system and run from the /persist directory.
```

Comment:
Would be good to add a description on how to cross-compile it.

Reply:
To be honest, I didn't think about cross-compiling. But it's a problem for the other binaries here as well. Good point. I would prefer to create a task and address it later.